ideascube/monitoring/forms.py

```diff
@@ -46,18 +46,19 @@ def clean_serials(self):
         return set([s for s in serials.splitlines() if s])
 
 
-class ExportEntryForm(forms.Form):
+class SinceForm(forms.Form):
 
-    since = forms.DateField(
-        widget=forms.DateInput(format='%Y-%m-%d'),
-        required=False)
+    since = forms.DateField(label=_('since'),
```

Comment:
See what I meant about confusion due to not using `verbose_name=` above? You've confused @vinyll :smiley: 

(that's of course compounded by the fact that Django has two very different things called "field")